File: app/src/main/java/com/krystianwsul/checkme/notifications/BootReceiver.java
package com.krystianwsul.checkme.notifications;

import android.annotation.SuppressLint;
import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

public class BootReceiver extends BroadcastReceiver {
    @SuppressLint("UnsafeProtectedBroadcastReceiver")
    @Override
    public void onReceive(Context context, Intent intent) {
        TickService.startServiceRegister(context, "BootReceiver: TickService.startServiceRegister");
    }
}

File: app/src/main/java/com/krystianwsul/checkme/notifications/BootReceiver.kt
package com.krystianwsul.checkme.notifications

import android.annotation.SuppressLint
import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

class BootReceiver : BroadcastReceiver() {

    @SuppressLint("UnsafeProtectedBroadcastReceiver")
    override fun onReceive(context: Context, intent: Intent) {
        TickJobIntentService.start(TickJobIntentService.getIntent(context, true, "BootReceiver"))
    }
}
